rdopkg_tar/tar_changes.py

```diff
@@ -123,6 +123,16 @@ def commit_distgit_amend(suffix):
     cmd = ['commit', '-a', '-F', '-', '--amend']
     git(*cmd, input=message, print_output=True)
 
+def check_gitlabuser():
+    """
+    Check for the original commiter username in jenkins env variables.
+    If it exists, then amend a suffix to the commit.
+    """
+    userName = os.environ.get('gitlabUserName')
+    gitlabuserName = os.environ.get('gitlabUserUsername')
+
+    if (userName and gitlabuserName):
+        commit_distgit_amend(suffix='GitLab-User: ' + gitlabuserName + ' (' + userName + ')')
```

Comment:
Please read https://www.codewithjason.com/dont-mix-refactorings-behavior-changes/

When refactoring, please preserve the old behavior *exactly*:
```suggestion
        commit_distgit_amend(suffix="GitLab-User: " + gitlabuserName + " " + userName)
```

Reply:
So would you like to change this format later in another MR? Or we can keep it this way.

Reply:
Would you like to make the change in another MR or we can preserve the old behavior?

Reply:
Let's preserve the old behavior for now.